### PR TITLE
ModalType prop added

### PR DESCRIPTION
**Pull request by HugoLaguna**
#### What problem is this solving?

The fragments of  orderForm/item hasn't modalType prop, we create a PR to have it in [checkout-graphql](https://github.com/vtex-apps/checkout-graphql/pull/175).

### CODE REVIEW

**Comment by HugoLaguna** [comment]
Patch (backwards-compatible bug fixes)

**Comment by HugoLaguna** [comment]
There's nothing new to document

**Review by jeffersontuc — DISMISSED** [review]

**Review by lucasecdb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [Unreleased]+
```
Comment (lucasecdb) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
```

**Review by lucasecdb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (HugoLaguna) [reply CHANGELOG.md]: fixed!

**Review by jeffersontuc — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**